CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ### VNEXT
 
-* ...
+* Merge schema now can accept resolvers in a plain object format [PR #511](https://github.com/apollographql/graphql-tools/pull/511)
```

Comment:
I think the main change is that `mergeInfo` is now on the `info` resolver argument, right?